File: backend/matlab/Tools/WriteOutTable.m
function WriteOutTable(table_out, table_in, second_table, third_table, out_name)
    if exist(out_name, 'file') == 2
        delete(out_name)
    end
    writetable(table_out, out_name, 'Sheet', 'Sheet1');
    writetable(table_in, out_name, 'Sheet', 'Sheet2');
    writetable(second_table, out_name, 'Sheet', 'Sheet3');
    writetable(third_table, out_name, 'Sheet', 'Sheet4');
end

File: backend/python/write_out_table.py
import pandas as pd
import os


def write_out_table(table_out, table_in, second_table, third_table, out_name):
    if os.path.exists(out_name):
        os.remove(out_name)

    with pd.ExcelWriter(out_name, engine='openpyxl') as writer:
        table_out.to_excel(writer, sheet_name='Sheet1', index=False)
        table_in.to_excel(writer, sheet_name='Sheet2', index=False)
        second_table.to_excel(writer, sheet_name='Sheet3', index=False)
        third_table.to_excel(writer, sheet_name='Sheet4', index=False)
